CHANGES.md

```diff
@@ -225,6 +232,8 @@ To be released.
     it fails to connect to all neighbors.  [[#933]]
  -  `Swarm<T>` became to respond to the messages with different app protocol version.
     [[#949]]
+ -  `BlockPolicy<T>` became to validate `Block<T>.StateRootHash` property
+     of a `Block<T>`.  [[#986]]
```

Comment:
Is this necessary despite we've never released `Block<T>.StateRootHash`?

Reply:
The reason why I added the line, was the behavior of `BlockPolicy<T>`. As you said, `Block<T>.StateRootHash` is had never released yet, but `BlockPolicy<T>` does. Then isn't it right to log the change or is it unnecessary?